src/client/app/components/unit/EditUnitModalComponent.tsx

```diff
@@ -410,7 +410,7 @@ export default function EditUnitModalComponent(props: EditUnitModalComponentProp
 										name='identifier'
 										type='text'
 										autoComplete='on'
-										onChange={e => { handleStringChange(e); }}
+										onChange={e => {handleStringChange(e);}}
```

Comment:
The changes in this file all appear to be formatting. The format changes are not what VSC does by default for js file. Given this, I think they all should be removed.